src/rgw/driver/rados/rgw_rest_log.cc

```diff
@@ -703,6 +709,8 @@ void RGWOp_DATALog_List::send_response() {
 
   s->formatter->open_object_section("log_entries");
   s->formatter->dump_string("marker", last_marker);
+  utime_t lu(last_update);
```

Comment:
the data sync changes look good! 
just one thing.. is there a reason why we need to encode this as utime_t?
we consume the response and do comparisons in real_time in `store_marker`